datalad/metadata/search.py

```diff
@@ -286,7 +286,10 @@ def _search_from_virgin_install(dataset, query):
                      "superdataset at %r?"
                      % DEFAULT_DATASET_PATH):
             from datalad.api import install
-            default_ds = install(DEFAULT_DATASET_PATH, source='///')
+            default_ds = install(
+                DEFAULT_DATASET_PATH,
+                source='///',
+                result_renderer='disabled')
```

Comment:
I actually think that in such cases it is ok to let commands render or we should `yield` up and render upstairs -- it is useful to communicate back what is going on/was done -- a large superdataset was installed.  So should either be "caught" and dataset extracted from result record and then re-yielded or could be left as it was IMHO

Reply:
Personally, I think always yielding is a good default.

Reply:
> 

True, true! Due to haste, that was left out.

Reply:
> "[...] always yielding is good"

I call `install` now with the `return_type='generator'` and disabled result rendering. All non-dataset results are yielded upwards.